ci-operator/step-registry/baremetalds/two-node/fencing/post-install/node-degredation/baremetalds-two-node-fencing-post-install-node-degredation-ref.yaml

```diff
@@ -0,0 +1,18 @@
+ref:
+  as: baremetalds-two-node-fencing-post-install-node-degredation
+  from: dev-scripts
```

Comment:
It's possible that the dev-scripts container doesn't have oc. Do we need the dev-scripts container? I know the pre-test steps container uses oc - so can we just use that container instead?

Reply:
See previous comment, I think it's safe to use `tests` instead of dev-scripts here